component/TodoTotalCount.js

```diff
@@ -0,0 +1,33 @@
+
+export function TodoTotalCount( $div, context ){
+    const $count = document.querySelector('.todo-count > strong');
+    const $filter = $div.querySelector('.filters');
+
+
+    this.setState = (todoItems) => {
+        this.render(todoItems);
+    };
+    
+    this.render = (todoItems) => {
+        $count.innerText = todoItems.length;
+        $filter.innerHTML = renderHTML();
+    };
+
+    $filter.addEventListener('click', e => {
+        const completeState = e.target.innerText;
+        context.filterTodo(completeState);
+    });
```

Comment:
현재 ```EventListener```에 콜백 함수를 등록할 때 대부분 인라인으로 함수를 작성하셨는데 별도의 함수로 분리 후 등록해주시면 가독성이 더욱 높아질거 같습니다!

```suggestion
const filterTodo = (e) => {
  const completeState = e.target.innerText;
  context.filterTodo(completeState);
}

$filter.addEventListener('click', filterTodo)
```

Reply:
이렇게 분리하는 방법도 있었군요! 피드백 감사합니다